### DISCUSSION TITLE
How does the `number-offset` option work?

### DISCUSSION THREAD

**Discussion by mariovagomarzal** [quarto-dev/quarto-cli#11065] [Q&A]
### Description

Hi everyone,

Could someone explain how the number-offset option works in the HTML format? I'm currently writing a book in web format using Quarto and I want to reset the chapter counter at the beginning of each part. I’ve been trying to set `number-offset: 0` in the first chapter of each part, but it doesn’t seem to have any effect. Any insights would be greatly appreciated!

**Comment by mcanouil** [comment]
See the relevant option in <https://quarto.org/docs/reference/formats/html.html#numbering> (for HTML).

**Reply by mariovagomarzal** [reply]
Yes, I've read it and after some more testing I've realized that it only affects the sections within a chapter and not the chapter itself.

To be more specific, I have a chapter structure as follows:
````qmd
chapters:
  - index.qmd
  - part: "Sample part"
     chapters:
        - foo.qmd
        - bar.qmd
  - part: "Other part"
     chapters:
        - other.qmd # <- Numbered as 3. I want to start the count on 1 again.
 ````

As I mentioned in the comment, the goal is to get that chapter numbered back to 1. Is there a way to do this?

**Reply by cderv** [reply]
`number-offset` is a Pandoc options. It can be set with Quarto but will mainly have expected Pandoc effect. 
Quarto Book is a format specific to Quarto and Pandoc does not handle anything after rendering each page. 
Chapters are handled by Quarto when processing all rendered pages to become a Book. 

I don't think we implemented something to change the numbering of chapter. This would be a feature request, and probably not an easy one. (as we would need to make that works for all formats HTML PDF, ...).

**Comment by mcanouil** [comment]
For your information:

- #5987
- #1283
